proxies/live/apiproxy/policies/Quota.General.xml

```diff
@@ -1,7 +1,8 @@
-<Quota name="Quota.General" type="rollingwindow">
+<Quota name="Quota.General" type="calendar">
     <Interval>1</Interval>
     <TimeUnit>minute</TimeUnit>
+    <StartTime>2022-7-26 12:00:00</StartTime>
```

Comment:
Any particular reason you didn't use the date from the JIRA? Not that I think it should make any difference. I assume if the TimeUnit is minute that it will only consider the second part.

Reply:
May be nice to keep it the same date as APIMs shared flow. Also the same date we are using in the referrals proxy under https://github.com/NHSDigital/e-referrals-service-api/pull/290/files#diff-b45e67487c4c9b7cc1578dff8b37e1dfd146d6531583d4eda5e61343f4af6d05

Reply:
Okay - NP